framework/encode/custom_vulkan_api_call_encoders.cpp

```diff
@@ -164,7 +164,7 @@ static void EncodeDescriptorUpdateTemplateInfo(VulkanCaptureManager*     manager
             {
                 size_t              offset = entry_info.offset + (entry_info.stride * i);
                 const VkBufferView* entry  = reinterpret_cast<const VkBufferView*>(bytes + offset);
-                encoder->EncodeHandleValue<BufferViewWrapper>(*entry);
+                encoder->EncodeHandleValue<vulkan_wrappers::BufferViewWrapper>(*entry);
```

Comment:
I wonder if there is another solution for delineating these Vulkan wrappers that doesn't require the `vulkan_wrappers::` prefix on every object. Some potential ideas:
1. Add a Vulkan-specific namespace (e.g., `gfxrecon::encode::vulkan` or `gfxrecon::encode::vk`) for all Vulkan-specific encode files. A Vulkan-specific namespace could also be added for `gfxrecon::decode`, e.g., `gfxrecon::decode::vulkan`
1. Adopt the DX12 wrapper naming pattern `[DX12 object name]_Wrapper`, so `vulkan_wrappers::BufferViewWrapper` becomes `VkBufferView_Wrapper`

Reply:
Your idea of the full name might work, but doing additional namespaces was much more involved than I wanted to go down.